src/System.Management.Automation/resources/RunspaceInit.resx

```diff
@@ -186,6 +186,9 @@
   <data name="NestedPromptLevelDescription" xml:space="preserve">
     <value>Dictates what type of prompt should be displayed for the current nesting level</value>
   </data>
+  <data name="NativeCommandUseErrorActionPreferenceDescription" xml:space="preserve">
+    <value>If true, $ErrorActionPreference applies to native executable non-zero exit codes</value>
```

Comment:
Could use some help with this wording.

Reply:
I think what you've got is pretty good -- hard to word. One alternative:

> If true, $ErrorActionPreference and -ErrorAction parameters will apply to native executables, so that non-zero exit codes will generate cmdlet-style errors governed by error action settings.

Reply:
@rjmholt Can you give an example of where the `-ErrorAction` parameter would come into play? I've tried `iex "whoami -xyzzy" -ea Stop` and `icm { whoami -xyzzy } -ea Stop`.  If I check the value of `$ErrorActionPreference` with the iex string or the icm scriptbock, the `-ErrorAction` parameter does not impact the value or pref variable.
```
PS> icm {
     "`$ErrorActionPreference is $ErrorActionPreference"
     $PSNativeCommandUseErrorActionPreference=$true
     whoami -xyzzy
} -ErrorAction Stop
$ErrorActionPreference is Continue
ERROR: Invalid argument/option - '-xyzzy'.
Type "WHOAMI /?" for usage.

NativeCommandException: Program "whoami.exe" ended with non-zero exit code 1.
```

Reply:
No you're right, there's no `-ErrorAction` parameter here. Just the preference variable

Reply:
Edited my suggestion above to correct it